src/test/fuzz/policy_estimator.cpp

```diff
@@ -48,7 +48,7 @@ FUZZ_TARGET(policy_estimator, .init = initialize_policy_estimator)
                 const auto tx_info = NewMempoolTransactionInfo(entry.GetSharedTx(), entry.GetFee(),
                                                                entry.GetTxSize(), entry.GetHeight(),
                                                                /* m_mempool_limit_bypassed */ false,
-                                                               /* m_submitted_in_package */ false,
```

Comment:
I'm not sure it is a good idea to have two calls to `fuzzed_data_provider` in the function call. The problem is that the order of evaluation of function arguments is unspecified: https://en.cppreference.com/w/cpp/language/eval_order
So evaluation order can change at any time. Then the same fuzzing input will produce different results.
I'd move the calls in front of `NewMempoolTransactionInfo`